hedera-node/docs/system-accounts-operations.md

```diff
@@ -0,0 +1,184 @@
+# System accounts
+
+## Definition
+The Hedera network reserves the first 1000 entity numbers for its own uses, and any account with a number in this reserved range is referred to as a system account.
+
+The following addresses are reserved for the Hedera precompiled contracts:
+- 0x167 (0.0.359) - HTS precompile;
+- 0x168 (0.0.360) - ExchangeRate precompile;
+- 0x169 (0.0.361) - PRNG precompile.
+
+Some of the system accounts exist today and more can be created in the future. Certain system accounts have predefined roles in the network. These include:
+- 0.0.2 - treasury account;
+- 0.0.50 - system admin;
+- 0.0.55 - address book admin;
+- 0.0.57 - exchange rates admin;
+- 0.0.58 - freeze admin;
+- 0.0.59 - system delete admin;
+- 0.0.60 - system undelete admin;
+- 0.0.800 - staking reward account;
+- 0.0.801 - node reward account.
+
+Addresses in range 1 → 9 are special addresses on which the Ethereum precompiled contracts exist. This is valid for both Ethereum and Hedera.
```

Comment:
BTW, that's the _current_ configuration of Ethereum.  This will expand in the future. (In fact `0x0A` through `0x13` are already reserved in BESU for EIP-2537 and over time there will be more.  

* (Also, to make matters more confusing, there is now with Cancun an EVM address for the new Ethereum "Beacon Root contract" - which is a contract, _not_ a precompile - that is special to Ethereum.  But this document needn't go in to it.)

Maybe also change "Addresses in range ..." to "EVM addresses in range ..."?

Reply:
Updated.